src/awsLambda.re

```diff
@@ -91,6 +91,34 @@ type handler('event, 'cb) =
 type authResponseContext = Js.Dict.t(any);
 
 module APIGatewayProxy = {
+  module Authorizer = {
+    type event = {
+      .
+      "type": string,
+      "methodArn": string,
+      "authorizationToken": string,
+    };
+    type statement = {
+      .
+      "Action": array(string),
+      "Effect": string,
+      "Resource": array(string),
+      "Sid": string,
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/aws-lambda/index.d.ts#L459

There is no `Sid` in the typescript types. Is this field documented somewhere?